File: learn-python(1~50)/10103.py
t = input()
score_1 = 100
score_2 = 100
for i in range(int(t)):
    a,b = map(int,input().split())
    if a>b:
        score_2 = score_2 - a
    if b>a:
        score_1 = score_1 - b
print(f'{score_1}\n{score_2}')

File: learn-python(1~50)/10103.c
#include <stdio.h>

int main() {
	int i,t;
	int score_1=100,score_2=100;
	int dice1,dice2;
	scanf("%d",&t);
	for(i=1;i<=t;i++) {
		scanf("%d%d",&dice1,&dice2);
		if (dice1>dice2)
		score_2-= dice1;
		if (dice2>dice1)
		score_1-= dice2;	
	}
	printf("%d\n%d",score_1,score_2);
}